core/src/test/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceServiceSpec.scala

```diff
@@ -853,4 +853,44 @@ class WorkspaceServiceSpec extends FlatSpec with ScalatestRouteTest with Matcher
     }
 
   }
+
```

Comment:
I think the only thing you need in `AttributeComponentSpec` is something that tests `findUniqueStringsByNameQuery`. It could use a check that makes sure you really are only getting unique strings from a collection that has duplicates as well as non-tag attributes.